### PR TITLE
[AKS] Remove syntax sugar for az aks update --ssh-access and update help message

### PR DESCRIPTION
**Pull request by norshtein**
Remove syntax sugar for az aks update --ssh-access and update help message

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

**Comment by norshtein** [comment]
@FumingZhang @zhoxing-ms @yanzhudd could you please have a review? Thanks!

File: src/aks-preview/azext_aks_preview/_params.py [hunk 1]
```diff
@@ -1415,8 +1415,6 @@ def load_arguments(self, _):
                 'For more information on "Auto" mode see aka.ms/aks/nap.'
             )
         )
-        # In update scenario, use emtpy str as default.
-        c.argument('ssh_access', arg_type=get_enum_type(ssh_accesses), is_preview=True)
```
Comment (FumingZhang) [inline src/aks-preview/azext_aks_preview/_params.py:1419]: Please leave a history note and mark it as a breaking change. Also release a new major version.

Comment (norshtein) [reply src/aks-preview/azext_aks_preview/_params.py:1419]: updated release note and bumped a new version.

**Review by FumingZhang — APPROVED** [review]
LGTM

Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=102901223&view=results) to validate the change.

- test_aks_disable_ssh

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**